README.md

```diff
@@ -897,6 +900,62 @@ QRCode qrCode = QRCode.create(
 QRCode qrCode = QRCode.getQRCode("external_id");
 ```
 
+### Customer
+
+#### Create Customer
+
+You can choose whether want to put the attributes as parameters or to put in inside a Map object.
+
+<table>
+<tr>
+<td>
+<pre>
+Customer.createCustomer(
+    String referenceId,
+    String mobileNumber,
+    String email,
+    String givenNames,
+    String middleName,
+    String surname,
+    String description,
+    String phoneNumber,
+    String nationality,
+    CustomerAddress[] addresses,
+    String dateOfBirth,
+    Map&lt;String, Object&gt; metadata
```

Comment:
new line

Reply:
new line gimana ko maksudnyaa? itu udah new line (?)

Reply:
```suggestion
    Map&lt;String,
    Object&gt; metadata
```

it's supposed to be like this right?